docs/pages/server-access/guides/vscode.mdx

```diff
@@ -10,8 +10,26 @@ This guide explains how to use Teleport and Visual Studio Code's remote SSH exte
 
 - [tsh client tool](https://goteleport.com/teleport/download) >= (=teleport.version=).
 - OpenSSH client.
-- Visual Studio Code with [remote SSH client](https://code.visualstudio.com/docs/remote/ssh#_system-requirements)
+- Visual Studio Code with a [remote SSH client](https://code.visualstudio.com/docs/remote/ssh#_system-requirements)
 for the Remote - SSH extension.
+- The Teleport Auth Service and Proxy Service, deployed on your own infrastructure or managed via Teleport Cloud.
+- One or more Teleport nodes with Server Access enabled. If you have not yet done this, read the [Server Access Getting Started Guide](../getting-started.mdx) to learn how.
+
+<Details 
+scopeOnly={true}
+scope={["oss", "enterprise"]}
+opened={false}
+title="Haven't deployed the Auth and Proxy Services?" >
+Follow one of our [getting started](../getting-started.mdx) guides to learn how to deploy the Teleport Auth Service and Proxy Service in your environment.
+</Details>
+
+<Details 
+scopeOnly={true}
+scope={["cloud"]}
+opened={false}
+title="Haven't deployed the Auth and Proxy Services?" >
```

Comment:
Is there a better title we can use here? With cloud, deploying services is not something you concern yourself with. Teleport takes care of that for you.